Add snippet about how to configure OIDC options (#11182)
aspnet/Docs: aspnetcore/security/authentication/azure-ad-b2c.md

> [!WARNING]
> Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in *appsettings.json*.

## Configuring the underlying OpenIdConnectOptions options

If you need to configure the OpenIdConnectOptions directly, you can configure them via:

> `serviceCollection.Configure<OpenIdConnectOptions>(AzureAD[B2C]Defaults.OpenIdScheme, o => { ... }`

Reviewer: There's also a JwtBearer version of this, right?

Author: Yeah i can tweak this to be more general, since it applies for cookies/jwtbearer as well

> [!WARNING]
> Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in *appsettings.json*.

> ## Configuring the underlying OpenIdConnectOptions/JwtBearer/Cookie options

Reviewer 2: ```suggestion
## Configure the underlying OpenIdConnectOptions/JwtBearer/Cookie options
```

> If you need to configure the OpenIdConnectOptions directly, you can configure them using the scheme constant:

Reviewer 2: ```suggestion
If you need to configure the <xref:Microsoft.AspNetCore.Authentication.OpenIdConnect.OpenIdConnectOptions> directly, you can configure them using the scheme constant:
```

> If you need to configure the OpenIdConnectOptions directly, you can configure them using the scheme constant:

Reviewer: ```suggestion
If you need to configure the underlying options directly, you can configure them using the scheme constant:
```
